blocks/i-lens/i-lens.js

```diff
@@ -114,26 +87,77 @@ function addStocksData(ilensBody, dropdown = 'default') {
   });
 }
 
-function addSecondDropDown(dropdownsDiv, dropdownValue) {
-  const ilensBody = dropdownsDiv.closest('.i-lens-container').querySelector('.i-lens-body');
+function createDropdown(ilensContainer, menuItems, dropDownDetails = []) {
+  // console.log(dropdownValue);
+  // const menuItems = dropdownValue.split(', ');
+  const dropdownText = menuItems[0].text;
+
+  const dropdownSelectDiv = document.createElement('div');
+  dropdownSelectDiv.className = 'dropdown-select';
+
+  const button = document.createElement('button');
+  button.className = 'dropdown-toggle';
+  button.innerHTML = `<span class="dropdown-text">${dropdownText}</span><span class="icon-down-arrow icon"></span>`;
```

Comment:
just to confirm that the value of `dropdownText ` is coming from json not from the page itself.

Reply:
@ravverma yes as confirmed from here https://github.com/aemsites/icicidirect/pull/205/files#diff-1d24df7cd7f8c1428271be3a80d890c3ba785bee6dededd50cdb60a2b98adfb4R93